src/extensions/default/HTMLHinter/parser.js

```diff
@@ -0,0 +1,52 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define*/
+
+define(function(require) {
+    "use strict";
+
+    var slowparse = require("../../../../node_modules/slowparse/slowparse.js");
```

Comment:
OK, leave this, add a comment about needing `npm install` from the root, and remove the config stuff below.